app/views/customers/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Customer Name: <%= @customer.name %></h1>
+
+<h1> Customer Items: </h1>
+
+<% @customer.items.each do |item| %>
+  <p>Item name: <%= item.name%></p>
+  <p>Price: <%= item.price%></p>
+  <p>Supermarket name: <%= item.supermarket_name%></p>
```

Comment:
Here is where it would be okay to do `item.supermarket.name` if you didn't want to make a separate model method (but it's not wrong to do so!)